File: Java/Euler1.java
public class Euler1
{
  public static void main(String[] args)
  {
    int total = 0;
    int endRange = 1000;
    for(int i = 0; i < endRange; i++)
    {
      if(i % 3 == 0 || i % 5 == 0)
      {
        total = total + i;
      }
    }
    System.out.println("The sum of numbers divisible by 3 and 5 that are less than 1000 is: " + total);
  }
}

File: Python/Euler1.py
total = 0
endRange = 1000
for x in range(0, endRange):
    if x % 3 == 0 or x % 5 == 0:
        total = total + x
print "The sum of numbers divisible by 3 and 5 that are less than 1000 is: %d " % total
